package.json

```diff
@@ -39,17 +39,16 @@
     "xmlhttprequest"
   ],
   "engines": {
-    "node": "^14.19.0 || ^16.14.0",
-    "yarn": "1.22.15"
+    "node": "^14.19.0 || ^16.14.0"
```

Comment:
@dimitropoulos Want to add Node 18 in here? It hit on the 19th.

Reply:
great call: did in 5be827dd7566d62ffa88787a9f01956c117635e8